Mathlib/GroupTheory/FreeAbelianGroupFinsupp.lean

```diff
@@ -73,7 +73,8 @@ theorem Finsupp.toFreeAbelianGroup_comp_toFinsupp :
 @[simp]
 theorem Finsupp.toFreeAbelianGroup_toFinsupp {X} (x : FreeAbelianGroup X) :
     Finsupp.toFreeAbelianGroup (FreeAbelianGroup.toFinsupp x) = x := by
-  rw [← AddMonoidHom.comp_apply, Finsupp.toFreeAbelianGroup_comp_toFinsupp, AddMonoidHom.id_apply]
+  rw [← AddMonoidHom.comp_apply, Finsupp.toFreeAbelianGroup_comp_toFinsupp,
+    AddMonoidHom.id_apply]
```

Comment:
Please review the entire PR for unnecessary formatting changes

Reply:
I don't know how to do that… Download the original file and diff?